src/middlewared/middlewared/plugins/chart_releases_linux/validation.py

```diff
@@ -191,6 +194,14 @@ async def validate_ix_mount_path(self, verrors, value, question, schema_name, re
         elif not validate_dataset_name(path):
             verrors.add(schema_name, f'Invalid dataset name {path}. "test1, ix-test, ix_test" are valid examples.')
 
+    @private
+    async def validate_acl_entries(self, verrors, value, question, schema_name, release_data):
+        try:
+            if value.get('path') and os.listdir(value['path']) and not value.get('options', {}).get('force'):
```

Comment:
2 problems:
1. blocking I/O in a coroutine
2. os.listdir just to see if there are contents is inappropriate